### PR TITLE
Update deps

### PR DESCRIPTION
**Pull request by patmaddox**

### CODE REVIEW

**Comment by patmaddox** [comment]
@jc00ke I am a bit skeptical that these updates are good, because while installing deps I saw some undefined function warnings related to crypto, similar as what we ran into the other day. They're in the Paseto library. Is that a part of code that just may not have enough test coverage?

**Comment by jc00ke** [comment]
https://github.com/RatioPBC/epi-contacts/blob/main/lib/epi_contacts/secure_id.ex
https://github.com/RatioPBC/epi-contacts/blob/main/test/epi_contacts/secure_id_test.exs

It's got test coverage, integration style. We may want to rev the Paseto version at some point from v2 to v4 or whatever the latest is.

**Comment by patmaddox** [comment]
> It's got test coverage, integration style. We may want to rev the Paseto version at some point from v2 to v4 or whatever the latest is.

Paseto is already at the latest (1.3.1) as of this PR.

**Comment by patmaddox** [comment]
Alright I've taken a closer look and it appears that the crypto issue only applies to v1 pasetos, and we use v2 pasetos.

File: lib/epi_contacts_web/views/search_view.ex [hunk 1]
```diff
@@ -10,8 +10,7 @@ defmodule EpiContactsWeb.SearchView do
```
Comment (jc00ke) [inline lib/epi_contacts_web/views/search_view.ex:10]: I don't think this function is even used anymore.

Comment (jc00ke) [reply lib/epi_contacts_web/views/search_view.ex:10]: It's not tested, that's for sure, and...

```
  ➜ rg search_criteria
  lib/epi_contacts_web/views/search_view.ex
  8:  def search_criteria(first_name, last_name, street, city, state, zip_code, phone) do
```

**Review by jc00ke — CHANGES_REQUESTED** [review]

File: lib/epi_contacts_web/views/search_view.ex (hunk 1 above)
Comment (jc00ke) [reply lib/epi_contacts_web/views/search_view.ex:10]: Maybe we save that for a later PR with [mix_unused](https://github.com/hauleth/mix_unused).

**Review by jc00ke — APPROVED** [review]

**Pull request merged**